eve/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+version: 0.2
+
+branches:
+  default:
+    stage: pre-merge
+
+stages:
+  pre-merge:
+    worker:
+      type: docker
+      path: eve/workers/unit_and_feature_tests
+      volumes:
+        - '/home/eve/workspace'
+    steps:
+      - Git:
+          name: fetch source
+          repourl: '%(prop:git_reference)s'
+          shallow: True
+          retryFetch: True
+          haltOnFailure: True
+      - ShellCommand:
+          name: run static analysis tools on markdown
+          command: npm run --silent lint_md
+      - ShellCommand:
+          name: run static analysis tools on code
+          command: npm run --silent lint
+      - ShellCommand:
+          name: run unit tests
+          command: npm test
+      - ShellCommand:
+          name: run feature tests
+          command: npm run ft_test
```

Comment:
what about the 'behavior' tests that are in the package.json ("bh_test": "mocha --recursive tests/behavior",)

Reply:
We were matching the circle enviroment so I'm not sure why those aren't included. @rahulreddy Should we add in the behavior tests into the CI?

Reply:
Yes we should.